src/globus_cli/exception_handling/hooks.py

```diff
@@ -515,6 +504,80 @@ def _handle_gare(gare: globus_sdk.gare.GARE, message: str | None = None) -> None
         )
 
 
+def _concrete_unauthorized_hook() -> None:
```

Comment:
What do you think about naming it as what it does rather than the hook it's implemented.

```suggestion
def _emit_unauthorized_message() -> None:
```

The hooks I assume are there with conventional names so that other locations can override them. But This is internal & won't ever be overridden so no utility in structuring it like that.

Reply:
Yeah, I'm happy with that rename.